contrib/pam_zfs_key/pam_zfs_key.c

```diff
@@ -642,10 +661,28 @@ pam_sm_authenticate(pam_handle_t *pamh, int flags,
 {
 	(void) flags, (void) argc, (void) argv;
 
+	zfs_key_config_t config;
+	zfs_key_config_load(pamh, &config, argc, argv);
```

Comment:
`zfs_key_config_load()` can fail, so either explicitly cast to `void` or handle the return value. I think the latter is better.

(There are two calls which ignore the return value somewhere else as well. Those are not related to your changes but I think it would be better to fix them too. )

@felixdoerre What do you think?

Reply:
Yes, it would be better to handle the error code in all occurrences.